packages/assets-controllers/src/TokenDetectionController.test.ts

```diff
@@ -12,10 +12,11 @@ import type {
   NetworkState,
   NetworkConfiguration,
   NetworkController,
-  ProviderConfig,
   NetworkClientId,
 } from '@metamask/network-controller';
 import { defaultState as defaultNetworkState } from '@metamask/network-controller';
+import type { AutoManagedNetworkClient } from '@metamask/network-controller/src/create-auto-managed-network-client';
```

Comment:
Yes I was surprised to learn that `AutoManagedNetworkClient` isn't a package-level export. Is this an intensional choice? Otherwise I could create a ticket for exporting this.

Reply:
It wasn't. I don't think I expected anyone to have to use this type, as it's low-level. But there's no harm in exporting it or anything.